<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>629</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>658</width>
    <height>629</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>658</width>
    <height>629</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Manuel Control</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
border-color: rgb(115, 210, 22);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="5" colspan="2">
    <widget class="QPushButton" name="down_button">
     <property name="text">
      <string>Down</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="2" colspan="2">
    <widget class="QPushButton" name="left_button">
     <property name="text">
      <string>Left</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3" colspan="2">
    <widget class="QPushButton" name="empty">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Speed</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2" colspan="2">
    <widget class="QPushButton" name="fw_button">
     <property name="text">
      <string>Forward</string>
     </property>
    </widget>
   </item>
   <item row="3" column="8">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Battery:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Speed:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="6" rowspan="2" colspan="2">
    <widget class="QPushButton" name="yawR_button">
     <property name="text">
      <string>YawRight</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" rowspan="2" colspan="2">
    <widget class="QPushButton" name="right_button">
     <property name="text">
      <string>Right</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QPushButton" name="bw_button">
     <property name="text">
      <string>Backward</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="10">
    <widget class="QLabel" name="camImg">
     <property name="minimumSize">
      <size>
       <width>640</width>
       <height>480</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>640</width>
       <height>480</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="5" rowspan="2" colspan="2">
    <widget class="QPushButton" name="up_button">
     <property name="text">
      <string>Up</string>
     </property>
    </widget>
   </item>
   <item row="4" column="8">
    <widget class="QPushButton" name="land_button">
     <property name="text">
      <string>Land</string>
     </property>
    </widget>
   </item>
   <item row="1" column="8" rowspan="2" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">color:rgb(204, 0, 0)</string>
     </property>
     <property name="text">
      <string>DRONE INFO</string>
     </property>
    </widget>
   </item>
   <item row="0" column="8" colspan="2">
    <widget class="QPushButton" name="closeM_button">
     <property name="text">
      <string>CLOSE</string>
     </property>
    </widget>
   </item>
   <item row="4" column="9">
    <widget class="QPushButton" name="takeoff_button">
     <property name="text">
      <string>Take Off</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="7">
    <widget class="QSlider" name="speed_slider">
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="pageStep">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="7">
    <widget class="QLabel" name="spd_label">
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="7">
    <widget class="QPushButton" name="stop_button">
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4" rowspan="2" colspan="2">
    <widget class="QPushButton" name="yawL_button">
     <property name="text">
      <string>YawLeft</string>
     </property>
    </widget>
   </item>
   <item row="3" column="9">
    <widget class="QLabel" name="btry_label">
     <property name="styleSheet">
      <string notr="true">color:rgb(0, 255, 0)</string>
     </property>
     <property name="text">
      <string>%100</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>speed_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spd_label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeM_button</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>587</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
